exercises/palindrome-products/.meta/src/reference/java/PalindromeCalculator.java

```diff
@@ -19,6 +21,9 @@
                 }
             }
         }
+        
+        if(palindromeSortedList.size() == 0) throw new NoSuchElementException("no palindrome with factors in the range " + minFactor + " to " + maxFactor);
```

Comment:
This would read more nicely if it was presented as:
```
if (palindromeSortedList.size() == 0)
    throw new NoSuchElementException("no palindrome with factors in the range " + minFactor + " to " + maxFactor);
```

Reply:
Please add curly brackets as well! 🙂 Otherwise it's easy to make mistakes (see https://stackoverflow.com/questions/8020228/is-it-ok-if-i-omit-curly-braces-in-java for more details) 🙂